conda/base/context.py

```diff
@@ -720,11 +720,23 @@ def channels(self):
         if self._argparse_args and 'channel' in self._argparse_args:
```

Comment:
This `if` block is necessary to cover the case in `test_specify_channels_cli_adding_defaults_no_condarc`.